pipeline_steps/multi_node_aio_prepare.groovy

```diff
@@ -46,11 +46,14 @@ def prepare() {
       common.prepareConfigs(
         deployment_type: "onmetal"
       )
-      sh """      
+      sh """
+      echo "multi_node_aio_prepare.prepare/Prepare RPC Configs"
+      set -xe
```

Comment:
https://jenkins.io/doc/pipeline/steps/workflow-durable-task-step/#sh-shell-script suggest to me that it shouldn't be necessary to set this here but I think that's only because there's no interpreter directive. The repo should be a bit more consistent on that.